scvi/model/_autozi.py

```diff
@@ -98,10 +102,15 @@ def __init__(
         beta_prior: Optional[float] = 0.5,
         minimal_dropout: float = 0.01,
         zero_inflation: str = "gene",
+        use_observed_lib_size: bool = True,
         **model_kwargs,
     ):
         super(AUTOZI, self).__init__(adata)
 
+        self.use_observed_lib_size = use_observed_lib_size
+        n_batch = self.summary_stats["n_batch"]
+        library_log_means, library_log_vars = _init_library_size(adata, n_batch)
+
         self.module = AutoZIVAE(
             n_input=self.summary_stats["n_vars"],
             n_batch=self.summary_stats["n_batch"],
```

Comment:
reuse prev defined `n_batch`? This happens in a few places I won't flag all of them